smoke-test.sh

```diff
@@ -293,13 +293,15 @@ function runAllTests() {
         doCommand C# classlib new restore build multi-rid-publish
         doCommand C# xunit new restore test
         doCommand C# mstest new restore test
+        doCommand C# worker new restore build run multi-rid-publish
```

Comment:
Should I omit `run` here?

Reply:
I'm fine with it, it looks like we're adding 90 seconds at most to the smoke-tests here.